tools/mpremote/mpremote/repl.py

```diff
@@ -2,13 +2,23 @@
 
 from .transport import TransportError
 
+import serial
```

Comment:
As above, suggest not relying on this package and catching `OSError` instead.

Reply:
They've all been cleaned up now thanks!